### PR TITLE
Improvements to mobile ui

### PR DESCRIPTION
**Pull request by griddoz92**
Improvements to mobile ui

### CODE REVIEW

**Comment by griddoz92** [comment]
Hi @kaso17 I have made some improvements to the mobile ui

**Comment by kaso17** [comment]
These changes break the layout on the desktop version for me.
Please make them conditional for mobile devices only.

**Comment by griddoz92** [comment]
fixed  the layout tested on both descktop and mobile

**Comment by kaso17** [comment]
It's still broken for me on the desktop:
https://nimbus.everhelper.me/client/notes/share/776391/607z9bhblo82dii3g5q4
https://nimbus.everhelper.me/client/notes/share/776392/f04vgbd7evdqm5lsr46h

I suggest you use CSS media queries to use alternative style blocks for mobile devices.

**Comment by griddoz92** [comment]
Can't seem to duplicate this one 
https://nimbus.everhelper.me/client/notes/share/776391/607z9bhblo82dii3g5q4
Which browser are you using

**Comment by griddoz92** [comment]
Can't seem to duplicate this one 
https://nimbus.everhelper.me/client/notes/share/776391/607z9bhblo82dii3g5q4
Which browser are you using

**Comment by kaso17** [comment]
I'm using firefox 51.0.1 on windows 7 with a 2550x1440 screen resolution.
I believe the main problem are your changes to "table td.fit". The class is supposed to keep columns as small as possible without wrapping the content on large screen systems (leaving as much space as possible for the torrent/indexer title). I can see how that might be a problem on mobile devices. But I don't see any other option besides using CSS media queries.

**Comment by griddoz92** [comment]
Ok that's why I couldn't duplicate it my pc doesn't have a high resolution screen

**Comment by griddoz92** [comment]
Fixed changed the configured indexers td class to actions and returned the td fit back to original format as the data tables in the modals are x axis scrolling

**Comment by kaso17** [comment]
https://picload.org/image/rlrpwldw/jackett.png 
* got some strange lowercase "l"s, no idea what's causing them yet. 
* action column really needs to be minimum with/nowrap on large screens
* cached releases and manual search modal have different width
* odd sizing behavior when changing tracker/category in manual search mode (will look into this, might be a JS issue)

**Comment by griddoz92** [comment]
Fixed all issues required please check for any other things I need to tweek

File: src/Jackett/Content/index.html [hunk 1]
```diff
@@ -392,7 +394,7 @@ <h4 class="modal-title">Manual search</h4>
                     <div class="modal-body">
                         <p>You can search all configured indexers from this screen.</p>
                         <label>Query</label>
-                        <input type="text" name="query" id="searchquery" style="width:400px" />
```
Comment (kaso17) [inline src/Jackett/Content/index.html]: I suggest this inline attribute is moved to the css files

File: src/Jackett/Content/index.html [hunk 2]
```diff
@@ -266,9 +268,9 @@
                     <tr class="unconfigured-indexer-row">
                         <td><a target="_blank" href="{{site_link}}" title="{{description}}">{{name}}</a></td>
                         <td>{{mains_cats}}</td>
-                        <td class="fit"><span title="{{type}}" class="type-{{type}}">{{type_icon_content}}</span></td>
```
Comment (kaso17) [inline src/Jackett/Content/index.html]: class="fit" should remain (this one and the one below). If you need a second class for mobile compatibility feel free to add it.

File: src/Jackett/Content/index.html [hunk 3]
```diff
@@ -32,7 +32,9 @@
 
     <link href="../bootstrap/bootstrap.min.css" rel="stylesheet">
     <link href="../animate.css" rel="stylesheet">
-    <link href="../custom.css" rel="stylesheet">
+    <link rel="stylesheet" href="custom.css" media="only screen and (min-device-width: 480px)">
+    <link rel="stylesheet" href="custom_mobile.css" media="only screen and (max-device-width: 480px)">
+
```
Comment (kaso17) [inline src/Jackett/Content/index.html]: I believe we should add a slash at the beginning of the href attributes

File: src/Jackett/Content/index.html (hunk 2 above)
Comment (griddoz92) [inline src/Jackett/Content/index.html]: Fixed

File: src/Jackett/Content/index.html (hunk 1 above)
Comment (griddoz92) [inline src/Jackett/Content/index.html]: Fixed

File: src/Jackett/Content/index.html [hunk 4]
```diff
@@ -215,7 +217,7 @@
                 <tbody>
                     {{#each indexers}}
                     <tr class="configured-indexer-row">
-                        <td><a target="_blank" href="{{site_link}}" title="{{description}}">{{name}}</a> <span title="{{type}}" class="type-{{type}}">{{type_icon_content}}</span></td>
+                        <td><a target="_blank" href="{{site_link}}" title="{{description}}">{{name}}</a> <span title="{{type}}" class="type-{{type}}">{{type_icon_content}}</span></td>l
```
Comment (kaso17) [inline src/Jackett/Content/index.html]: There it it. Please remove the "1" at the end of the line.

**Comment by griddoz92** [comment]
Fixed

**Comment by kaso17** [comment]
merged, thank you

**Pull request merged**